### PR TITLE
Fix/configuration load

### LINKED ISSUE #19: Exception when loading configuration with no app.config settings

### PR DESCRIPTION
**Pull request by fireout**
Fixes #19, if no configuration is defined, GetConfigurationPath will return null.
Also renamed namespace, to remove WordSequence, should now all be Sequencer.

### CODE REVIEW

**Comment by fritzophrenic** [comment]
Some of the *.Designer.cs files still have WordSequence namespace uses.

Also the ItemGenerator/ItemVisitor files but those don't seem to be used right now.

Other than that, the changes look pretty good/complete, although I didn't try them out at all. I assume you just made this a pull request to give me a chance to look at it, but let me know if you wanted me to do more than that.

**Comment by fireout** [comment]
Yes, I wanted for the changes to be seen and minimally reviewed, thanks for your comments btw.

I did the changes in the Designer files, but I want to retest everything before pushing all the corrections, and I'm not sure if I can since there is a merge conflicts...

I removed any ItemVisitor/ItemGenerator files, they were supposed to be in the IOC branch, they are changes I had started before migrating to git.

Bottom line, I will push those changes soon, and make this version the develop branch.  I don't have a very precise release schedule, but I would love to have a "version" release (beta maybe) when we will have most of the issue complete, just so we can have a working master branch.

I'll try to work on the milestones just so it is clear.
